charts/flink-job/templates/flinkjobcluster.yaml

```diff
@@ -70,6 +70,10 @@ spec:
       {{- with .Values.podLabels }}
       {{- toYaml . | nindent 6 }}
       {{- end }}
+    {{- with .Values.affinity }}
```

Comment:
@eugenend Can you confirm that this key does exist in the Flink CRD?

Reply:
Yes, I'll close this PR for now